ms_db/ms_db.http

```diff
@@ -0,0 +1,6 @@
+@ms_db_HostAddress = http://localhost:5175
+
+GET {{ms_db_HostAddress}}/weatherforecast/
```

Comment:
Why are you getting the weatherforecast c# app?

Reply:
This is the skeleton when building with dotnet add webapi -controller
Fixed it